### ISSUE TITLE
🔥 Python 2 is dead! 🔥

### ISSUE THREAD

**Issue by micxer** [jfrog/docker2artifactory#22]
The requirements state that this tool runs with Python 2 yet Python 2 is dead since April. Does this work unmodified with a current Python version? Shoudn't this be updated as this tool will be useful in the future as well?

**Comment by jfrog-aparicio** [comment]
Yes. There are plans to rewrite the tool in a different language. As for Python 2 being dead, that does not prevent the tool from working in its current state. You may not want to install Python 2 but it is easy enough to run a Docker container version of it (as provided in the instructions) to avoid installing the deprecated version of Python.
